### ISSUE TITLE
Rename EMARDD Syndrome to MEGF10-Related Myopathy

### ISSUE THREAD

**Issue by RedRiver1816** [monarch-initiative/mondo#1112]
**Mondo term (ID and Label):** Early-onset myopathy-areflexia-respiratory distress-dysphagia syndrome (MONDO:0013731)

**Suggested new label:** MEGF10-Related Myopathy, MEGF10 Myopathy

**_Optional_: Any additional information (like supporting evidence, PubMed ID, etc.)** The ClinGen Congenital Myopathy GCEP reviewed this classification and determined this naming schema does not reflect the phenotypes of all probands with variants in MEGF10, particularly as a later-onset milder variant is reflected under the same disease entity (PMIDs: 22371254, 27460346 for examples). These phenotypes also appear individually or in various combinations in other myopathies (Hereditary myopathy with early respiratory failure PMID: 15802564, TPM2-Related Myopathy (Historically Nemaline/Cap Myopathy) PMID: 19047562) which could be confusing for clinicians.

@nicolevasilevsky : This is a ClinGen request

**Comment by nicolevasilevsky** [comment]
Thanks @RedRiver1816, I revised the label.

I added this text definition as well, does this look okay?

A congenital myopathy caused by mutations in the multiple epidermal growth factor-like domains 10 (MEGF10) gene, which causes early-onset myopathy characterized by severe weakness, scoliosis, joint contractures, areflexia, respiratory distress, and dysphagia, and a milder phenotype of minicore myopathy.
